radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -172,6 +172,22 @@ elseif(PCB STREQUAL X7)
     add_definitions(-DRADIO_FAMILY_JUMPER_T12)
     add_definitions(-DEEPROM_VARIANT=0x4003)
     add_definitions(-DMANUFACTURER_JUMPER)
+  elseif(PCBREV STREQUAL LR3PRO)
+    set(DEFAULT_INTERNAL_MODULE CROSSFIRE CACHE STRING "Default internal module")
+    set(INTERNAL_MODULE_SERIAL YES)
+    set(FLAVOUR lr3pro)
+    set(NAVIGATION_TYPE 9x)
+    set(CPU_TYPE STM32F2)
+    set(CPU_TYPE_FULL STM32F205xE)  # for size report
+    add_definitions(-DSTM32F205xx)
+    set(LINKER_SCRIPT targets/taranis/stm32f2_flash.ld)
+    set(LUA_EXPORT lua_export_lr3pro)
+    add_definitions(-DRADIO_LR3PRO)
+    add_definitions(-DRADIO_FAMILY_JUMPER_T12)
+    add_definitions(-DEEPROM_VARIANT=0x4003)
```

Comment:
No `EEPROM_VARIANT` needed, new radios should support YAML only, no binary format.